quic/s2n-quic-core/src/recovery/bbr.rs

```diff
@@ -1053,3 +1052,17 @@ impl BbrCongestionController {
         !self.try_fast_path || model_updated || prev_min_rtt != self.data_volume_model.min_rtt()
     }
 }
+
+#[derive(Debug, Default)]
+pub struct Endpoint {}
```

Comment:
Not a blocker here but before this is released, this should be either non-exhaustive or include a private field so we don't break consumers if we add configuration fields.

Reply:
Good point, I've gone ahead and added it now to BBR and Cubic